src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -25,6 +25,12 @@ object FireCloudConfig {
     lazy val methodsBaseUrl = authUrl + methodsPath
     lazy val configurationsPath = methods.getString("configurationsPath")
     lazy val configurationsBaseUrl = authUrl + configurationsPath
+
+    def methodsPermissionsFromMethod(namespace:String, name:String, snapshotId:Int) = authUrl +
```

Comment:
Suggestion: shorten name. `methodsPermissions` is explanatory enough without being redundant.